src/jdk.compiler/share/classes/com/sun/tools/javac/code/Source.java

```diff
@@ -238,6 +238,7 @@ public enum Feature {
         UNCONDITIONAL_PATTERN_IN_INSTANCEOF(JDK21, Fragments.FeatureUnconditionalPatternsInInstanceof, DiagKind.PLURAL),
         RECORD_PATTERNS(JDK21, Fragments.FeatureDeconstructionPatterns, DiagKind.PLURAL),
         STRING_TEMPLATES(JDK21, Fragments.FeatureStringTemplates, DiagKind.PLURAL),
+        SUPER_INIT(JDK21, Fragments.FeatureSuperInit, DiagKind.NORMAL),
```

Comment:
Now JDK mainline is 22, this should update the target version too.

Reply:
Thanks - should be fixed in a5f8cc5e8ff.